profiles/fscloud/README.md

```diff
@@ -0,0 +1,6 @@
+# Financial Services Cloud Profile
+
+This is a profile for COS that meets FS Cloud requirements. This profile assumes you are deploying into an already compliant account.
+It has been scanned by [IBM Code Risk Analyzer (CRA)](https://cloud.ibm.com/docs/code-risk-analyzer-cli-plugin?topic=code-risk-analyzer-cli-plugin-cra-cli-plugin#terraform-command) and meets all applicable goals.
```

Comment:
Well this is not quite true. There are 2 known failures that CRA produce:

`3000107 - Check whether Cloud Object Storage network access is restricted to a specific IP range`
`3000116 - Check whether Cloud Object Storage bucket resiliency is set to cross region`

Maybe we list those here and explain that we use CBRs for allowlisting, and we use bucket replication for resiliency?